### PR TITLE
feat: initial commit

### PR DESCRIPTION
**Pull request by bjavaid**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  filters: {
+    "bluebase.boot.end": (bootOptions: BootOptions, _ctx: any, BB) => {
+      const url = BB.Configs.getValue("plugin.sentry.url");
+      if (url !== null) {
+        Raven.config(url).install();
+      }
+      return bootOptions;
+    },
+    "bluebase.logger.log": (message: string, data: any) => {
+
```
Comment (artalat) [inline src/index.ts]: implementation missing here?

Comment (danial24) [reply src/index.ts]: added manual Capturing

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  filters: {
+    "bluebase.boot.end": (bootOptions: BootOptions, _ctx: any, BB) => {
+      const url = BB.Configs.getValue("plugin.sentry.url");
+      if (url !== null) {
+        Raven.config(url).install();
+      }
+      return bootOptions;
+    },
+    "bluebase.logger.log": (message: string, data: any) => {
+      Raven.captureMessage(message, data);
```
Comment (artalat) [inline src/index.ts]: there is no way to capture severity?

Comment (danial24) [reply src/index.ts]: let me check it.

Comment (danial24) [reply src/index.ts]: You can tag the captured message.
By default there is a param sent named level and there is error in it

Added a logger level param which will track the severity

The logger level tag sent is also filterable on sentry portal, you can group the messages with certain logger level. Tested it on Sentry as well.

**Comment by artalat** [comment]
Looks good to go. Merging it in develop now. But please create a separate PR to add boilerplate code from `bluebase plugin:create` command. That will add continuous integration and semantic releases to the project.

**Pull request merged**